.gitbook.yaml

```diff
@@ -16,4 +16,5 @@ redirects:
   getting-started/deploying-zenml/manage-the-deployed-services/troubleshoot-your-deployed-server.md: how-to/manage-the-zenml-server/troubleshoot-your-deployed-server.md
   how-to/stack-deployment/implement-a-custom-integration.md: how-to/contribute-to-zenml/implement-a-custom-integration.md
 
-  getting-started/zenml-pro/system-architectures: getting-started/system-architectures.md
\ No newline at end of file
+  getting-started/zenml-pro/system-architectures: getting-started/system-architectures.md
+  how-to/build-pipelines/name-your-pipeline-and-runs.md: how-to/build-pipelines/name-your-pipeline-runs.md
```

Comment:
```suggestion
  how-to/build-pipelines/name-your-pipeline-and-runs: how-to/build-pipelines/name-your-pipeline-runs.md
```

Reply:
@schustmi forgot this

Reply:
@htahir1 Are you sure? All the other lines in this file have the md suffix, only this one is missing? I think this might have been a file that was without suffix, or someone put a wrong entry into this redirect section.

Reply:
@schustmi hundred percent sure. if that file is like that we need to fix it ASAP :-) https://docs.gitbook.com/integrations/git-sync/content-configuration#redirects

Reply:
fixed